frontend/blog-angular-ui/src/app/welcome/welcome.component.html

```
@@ -1,14 +1,15 @@
   you can mange your posts <a routerLink="/posts">here</a>
 </div>
 
 <div class="container">
   <button (click)="getWelcomeMessage()" class="btn btn-success">Get Welcome Message</button>
 </div>
 
-<div *ngIf="welcomeMessage" class="container">
-  <h2> Your customized welcome -> {{ welcomeMessage }} </h2>
+@if (welcomeMessage()) {
+<div class="container">
+  <h2>Your customized welcome -> {{ welcomeMessage() }}</h2>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)